examples/axes_grid1/simple_axes_divider1.py

```diff
@@ -18,8 +18,8 @@ def label_axes(ax, text):
                    left=False, labelleft=False)
 
 
-fig = plt.figure(figsize=(12, 6))
-sfs = fig.subfigures(1, 2)
+fig = plt.figure(figsize=(6, 12))
+sfs = fig.subfigures(2, 1)
```

Comment:
The `Size.Fixed` is maybe too small now in this example?

Reply:
... also perhaps useful to make the figure have a facecolor here to make it clear how large the figure is?